pipeline-templates/job/code-build.yml

```diff
@@ -23,25 +24,25 @@ jobs:
     - template: azure-pipelines-templates/build/step/dependency-check.yml@das-platform-building-blocks
 
     - task: DotNetCoreCLI@2
-      displayName: Publish - dotnet publish ${{ SolutionBaseName }}
+      displayName: "Publish $(SolutionBaseName)"
       inputs:
         command: publish
         publishWebProjects: false #set to false as this setting (which defaults to true) will parse the entire repo for web projects
-        projects: src/${{ SolutionBaseName }}/${{ SolutionBaseName }}.csproj
-        arguments: -o $(build.artifactstagingdirectory)/publish -c ${{ BuildConfiguration }} --no-build
+        projects: src/$(SolutionBaseName)/$(SolutionBaseName).csproj

```

Comment:
das-github-template has SolutionBaseName passed as a parameter from azure-pipelines.yml for explicitness. If conventions should change, there should be a PR to das-github-template.